dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -3267,7 +3267,7 @@ paths:
         schema:
           type: "array"
           items:
-            $ref: "#/definitions/SourceFile"
```

Comment:
Sigh

Reply:
:sob:

Reply:
is this deterministically generated as `type: "object"` now?

Reply:
Don't know.
One other oddity,  editHostedTool and editHostedWorkflow got generated in different ways.